_sass/jekyll-theme-isotc211.scss

```diff
@@ -6,7 +6,7 @@ $widescreenBreakpoint: 1200px;
 $logoSize: 100px;
 $logoOffset: $logoSize + 12px;
 
-$isoTCColor: #0f6db3;
+$isoTCColor: #0f6db3 !default;
```

Comment:
Perhaps we could add a file like `_sass/config.scss`, which would be [`@import`](https://sass-lang.com/documentation/at-rules/import)-ed in stylesheets, and which would be a good place for setting overrides for variables like this one? The file would be empty in `geolexica-server` (no overrides for obvious reasons), but sites like one for IEV could provide its own version of that file in order to inject its own colours?

@strogonoff what do you think?